internal/models/token.go

```diff
@@ -9,6 +9,17 @@ import (
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 )
 
+type KeycloakToken struct {
```

Comment:
Maybe not something to do in this PR unless you think it makes sense, but it feels like this will be easy to conflate with the `Token` below, which is the internal/registration token that we mint vs a Keycloak token. It might be worth either naming our `Token` model differently, or having at least having a separate model for the Keycloak token? Also, this is the entire Keycloak token response, not just a specific token since it includes the refresh and access tokens, as well as the other metadata, so it might be worth naming it something like `KeycloakTokensObject` or `KeycloakTokensResponse`?